BarcodeParserBuilder/Infrastructure/AimParser.cs

```diff
@@ -59,6 +60,31 @@ internal static string StripBarcodePrefix(string barcodeString)
             return barcodeString[3..];
         }
 
+        internal static bool HasAimIdentifier(string barcodeString)
+        {
+            return barcodeString.Length > 3 && barcodeString.StartsWith("]");
+        }
+
+        /// <summary>
+        /// AIM identifier consist of code and modifier. AIM code defines the barcode type, for example EAN, CODE39, GS1 etc
+        /// </summary>
+        /// <param name="barcodeString">raw string from the reader</param>
+        /// <returns></returns>
+        internal static string GetAimIdentifierCode(string barcodeString)
```

Comment:
why not create a class called AimInformation and just return that info in 1 go? no need to call both functions if you always want the full information anyway